projects/Mallard/src/hooks/__tests__/use-edition-provider.altlocale.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+    defaultEditionCache,
+    editionsListCache,
+    selectedEditionCache,
+} from 'src/helpers/storage'
+import { BASE_EDITION, defaultEditionDecider } from '../use-edition-provider'
+
+jest.mock('src/services/remote-config', () => ({
+    remoteConfigService: {
+        getBoolean: jest.fn().mockReturnValue(true),
+    },
+}))
+
+jest.mock('src/helpers/locale', () => ({
+    locale: 'fr_FR',
+}))
+
+describe('useEditions', () => {
+    describe('defaultEditionDecider', () => {
+        beforeEach(async () => {
+            await editionsListCache.reset()
+        })
+        it('should set the BASE EDITION if remote config is on but locale is not in the list', async () => {
```

Comment:
It would be good to write few more tests with different configurations

Reply:
The tests written actually cover all the conditions with the chance in the locale mock. We have those that match the locale and those that dont. We can add more if it needs to be more explicit?